packages/web-components/src/components/rux-breadcrumb/rux-breadcrumb-item/rux-breadcrumb-item.tsx

```diff
@@ -0,0 +1,75 @@
+import { Component, h, Host, Prop, Element } from '@stencil/core'
+
+/**
+ *slots and parts
+ */
+
+/**
+ * @part container - the li containing the link element
+ * @part link - the link element
+ */
+
+@Component({
+    tag: 'rux-breadcrumb-item',
+    styleUrl: 'rux-breadcrumb-item.scss',
+    shadow: true,
+})
+export class RuxBreadcrumbItem {
+    // private breadcrumbParent: HTMLRuxBreadcrumbElement | null = null
+    @Element() el!: HTMLRuxBreadcrumbItemElement
+    /**
+     * The href property of the breadcrumb item link
+     */
+    @Prop({ reflect: true }) href?: string
+
+    /**
+     * The current item in the breadcrumb trail
+     */
+    @Prop({ reflect: true }) current?: boolean = false
+
+    connectedCallback() {
+        this._handleSlotChange = this._handleSlotChange.bind(this)
+    }
+
+    componentWillLoad() {
+        this._handleSlotChange()
+    }
+
+    private _handleSlotChange() {
+        const slot = this.el?.shadowRoot?.querySelector(
+            'slot'
+        ) as HTMLSlotElement
+        //if the slot exists
+        if (slot) {
+            const link = this.el.shadowRoot?.querySelector(
+                '[part="link"]'
+            ) as HTMLElement
+            const fontSize = window
+                .getComputedStyle(link)
+                .getPropertyValue('font-size')
+            const childEls = slot.assignedNodes({}) as HTMLElement[]
+            //slotted content might just be text which would return nothing
+            if (childEls.length) {
+                //give rux-icons the correct size
+                childEls.map((el) => {
+                    if (el.nodeName === 'RUX-ICON')
+                        el.setAttribute('size', fontSize)
```

Comment:
![Screen Shot 2023-01-12 at 10 55 47 AM](https://user-images.githubusercontent.com/1091342/212115794-78133ecb-a1f0-448a-9928-297c2f483ccd.png)

should the indicator size also change/scale with font size?

Reply:
I think that is. nice to have but probably bears more discussion. Looking at the way we would have to do that in the current iteration could be tricky (what if breadcrumb-items have different font sizes.. where do we decide to pull the size from?.. do we make them all one size or dependent on their preceding breadcrumb-item?)

If we DO want to do this for the MVP I think it's possible though. We just need a quick chat about it first.

Reply:
for MVP I think we can table this for now